text/0001-workflow.md

```diff
@@ -29,32 +29,21 @@ The are three kinds of RFCs:
 * `withdrawn`: the RFC was withdrawn.  Typically such RFCs are not visible in the repository as the corresponding PRs are not merged unless they are withdrawn after accepted.
 * `replaced`: the RFC was later replaced by another RFC.
 
-# RFC Creation Workflow
+# Workflow
 
-1. Create a new RFC document as a copy of `0000-template.md`.
-2. Name it `XXXX-short-description.md` and commit it to a new branch.
-3. Create a pull request against the `rfcs` repository.  The number of the pull request then
-   becomes the assigned RFC number filled into `XXXX`.  Zero pad it out to 4 places for better sorting.
-4. Pick an RFC type and write it down on your RFC text into the header.
+1. Create a new RFC document as a copy of `0000-template.md`. Name it `XXXX-short-description.md` and commit it directly to the `main` branch. Anyone can do this at any time.
+1. Modify your RFC directly on `main` or through PRs as you see fit, except that ...
+1. For non-informational RFCs, updating the RFC status to `approved` must happen in a PR, with relevant parties signing off.
```

Comment:
For RFCs that are made on main, how would the 'comments' be done? One strength of having the entire RFC show up as a pull request is the workflow that github provides around providing feedback and incorporating that feedback.

Reply:
I guess the mental model is: publishing on main is the "request", and subsequent PRs are the "comments." How about this?

```suggestion
1. Modify your RFC directly on `main` until other people care about it, then switch to PRs to track comments and changes.
1. For non-informational RFCs, updating the RFC status to `approved` _must_ happen in a PR, with relevant parties signing off.
```

Reply:
> subsequent PRs

... or Issues, or Discussions, or Discord, or Discord Forums, or ... 😭

Reply:
Right now PRs that add a new RFC are requests for comments about requests for comments. I propose we unnest. :)